.github/renovate.json5

```diff
@@ -191,6 +193,7 @@
       "allowedVersions": "22.04",
       "matchBaseBranches": [
         "main",
+        "v1.14",
```

Comment:
Can't comment on the actual line, but I think we'll need to add a section to restrict Go toolchain to the currently used 1.20 in `1.14` and avoid upgrading to 1.21 once it's out:

```json
    {
      "matchPackageNames": [
        "docker.io/library/golang",
        "go"
      ],
      "allowedVersions": "<1.21",
      "matchBaseBranches": [
        "v1.14"
      ]
    },
```

Reply:
Added.

Reply:
I've also added similar logic for the alpine image bumps.